maintainers/maintainer-list.nix

```diff
@@ -2,18 +2,18 @@
   List of NixOS maintainers.
    ```nix
    handle = {
-     # Required
-     name = "Your name";
+   # Required
+   name = "Your name";
 
-     # Optional, but at least one of email, matrix or githubId must be given
-     email = "address@example.org";
-     matrix = "@user:example.org";
-     github = "GithubUsername";
-     githubId = your-github-id;
+   # Optional, but at least one of email, matrix or githubId must be given
+   email = "address@example.org";
+   matrix = "@user:example.org";
+   github = "GithubUsername";
+   githubId = your-github-id;
 
-     keys = [{
+   keys = [{
        fingerprint = "AAAA BBBB CCCC DDDD EEEE  FFFF 0000 1111 2222 3333";
-     }];
+   }];
```

Comment:
Unrelated whitespace changes: please revert this to the previous indentation.

Maybe you have a "format on save" tool that is misbehaving?

Note this is part of your [maintainers: add m0ustach3](https://github.com/NixOS/nixpkgs/pull/437310/commits/403154ff68d67fee3b18088a9813a0aaa33a89ea) commit, so _that_ commit should be amended to remove the unrelated changes. If you're not careful, you could end up adding the changes in one commit, and reverting them in the next 😁

Reply:
Fair enough.